### ISSUE TITLE
Garage status updates

### ISSUE THREAD

**Issue by c650** [cs-discord-at-ucf/lion#665]
Periodically poll the garage statuses and alert when a garage starts to fill up 

A simple version can report all garages that are over 90% full 

a little better is keeping a recent history (don’t repeat alerts too often) 

Stretch:
even better is calculating a fill rate of each garage and watch to see when that rate increases

**Comment by c650** [comment]
I specifically want someone new to the team to do this.

**Comment by pedrofcarvalho** [comment]
being new here, I will probably do this one then

I will be dropping some questions about it whenever I get to it

**Comment by tanndlin** [comment]
tbh the data that UCF gives on the garages is like *always* wrong, so this could be not so useful at times

**Comment by pedrofcarvalho** [comment]
yep, I have to agree. For POOSP we decided to do an ML project with this data and it was....bad lol
